backend/src/controllers/seasons/seasons.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Get, Body, Post } from "@nestjs/common";
+import { ApiUseTags, ApiResponse } from "@nestjs/swagger";
+import { SeasonDto } from "src/models/season.dto";
+import { SeasonsService } from "src/services/seasons.service";
+
+@ApiUseTags("Seasons")
+@Controller("api/seasons")
+export class SeasonsController {
+  constructor(private seasonsService: SeasonsService) {}
+
+  @ApiResponse({
+    status: 200,
+    description: "Returns seasons",
+    isArray: true,
+    type: SeasonDto,
+  })
+  @Get()
+  async listAll(): Promise<SeasonDto[]> {
+    return this.seasonsService.all();
+  }
+
+  @Get("/currentSeason")
```

Comment:
Just a small comment but can we keep the urls in kebab-case? Or maybe just change the url to /current since it's already under /seasons (/seasons/current).

Reply:
Fixed!